editor/src/components/editor/persistence.ts

```diff
@@ -292,9 +293,14 @@ export async function triggerForkProject(
     ...editor,
     forkedFromProjectId: oldProjectId,
     id: newProjectId,
+    name: editor.projectName + ' (forked)',
```

Comment:
```suggestion
    name: `${editor.projectName} (forked)`,
```

this is just a coding style nitpick